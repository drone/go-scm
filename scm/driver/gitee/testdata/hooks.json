[
  {
    "id": 787341,
    "url": "http://test.kit101.com/webhook",
    "created_at": "2021-10-07T10:27:26+08:00",
    "password": "123asdas123",
    "project_id": 14836026,
    "result": "\u003Chtml\u003E \u003Chead\u003E\u003Ctitle\u003E403 Forbidden\u003C/title\u003E\u003C/head\u003E \u003Cbody\u003E \u003Ccenter\u003E\u003Ch1\u003E403 Forbidden\u003C/h1\u003E\u003C/center\u003E \u003C/body\u003E \u003C/html\u003E",
    "result_code": 403,
    "push_events": true,
    "tag_push_events": true,
    "issues_events": true,
    "note_events": true,
    "merge_requests_events": true
  }
]