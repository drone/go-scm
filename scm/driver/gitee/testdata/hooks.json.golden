[
  {
    "ID": "787341",
    "Name": "",
    "Target": "http://test.kit101.com/webhook",
    "Events": [
      "push",
      "tag_push",
      "issues",
      "note",
      "merge_requests"
    ],
    "Active": true,
    "SkipVerify": true
  }
]